doc/development.md

```diff
@@ -0,0 +1,209 @@
+But first we'll set up _Synapse_ and build a configuration for them. 
+
+#### Create a _Synapse_ configuration
+We'll set up _Synapse_ in a way that's it will store the database files as well as the configuration files in a local data directory.
+Having it local makes it much easier to do changes on it.
+
+Run the following commands in a directory of your choice!
+
+Create a `data` directory.
+```bash
+mkdir data
+```
+As next let us create a default _Synapse_ configuration in `data` folder.
+```shell
+docker run -it --rm \
+    -v $(pwd)/data:/data \
+    -e SYNAPSE_SERVER_NAME=localhost \
+    -e SYNAPSE_REPORT_STATS=no \
+    matrixdotorg/synapse:latest generate
+```
+
+#### Customize the _Synapse_ configuration
+Let us customize the `data/homeserver.yml` configuration a little.
+This will make our live easier in local development process:
+
+**Activate unsecure registration**: Its help us to register users easier:
+For this add the following lines in the `data/homeserver.yml`
+```yaml
+enable_registration: true
+enable_registration_without_verification: true
+```
+
+**Deactivate Reporting** by adding this:
+```yaml
+report_stats: false
+```
+
+**Increase the Rate Limit**: Because we need to do sometime load testing, it is a good thing that's _Synapse_ allows massive logins from the same IP.
+For this add the following lines in the `data/homeserver.yml`
+```yaml
+rc_login:
+  address:
+    per_second:  15
+    burst_count: 5
+  account:
+    per_second:  18
+    burst_count: 4
+  failed_attempts:
+    per_second: 19
+    burst_count: 7
+```
+
+#### Run the Homeserver
+
+```shell
+docker run -d --name shadowfax-synapse \
+    -v $(pwd)/data:/data \
+    -p 8008:8008 \
+    matrixdotorg/synapse:latest
+```
+
+#### Create the SFU user and some other
+We need to create the SFU user in _Synapse_.
+
+```shell
+docker exec -it shadowfax-synapse register_new_matrix_user http://localhost:8008 -c /data/homeserver.yaml -u sfu  -p sfu --no-admin
+```
+```shell
+docker exec -it shadowfax-synapse register_new_matrix_user http://localhost:8008 -c /data/homeserver.yaml -u user1 -p user1 --no-admin
+```
+```shell
+docker exec -it shadowfax-synapse register_new_matrix_user http://localhost:8008 -c /data/homeserver.yaml -u admin -p admin --admin
+```
+
+#### Run Admin Tool (optional)
+
+Sometimes it is nice to see what happen in the Homeserver. 
+For this you can use this tool and the admin user we created a step previous to login.
+
+```shell
+docker run -d -p 8081:80 awesometechnologies/synapse-admin
+```
+
+---
+
+### Start the Frontend
+_Element Call_ is the frontend component. 
+The main part of the WebRTC logic you will find is in the [Matrix JS SDK](https://github.com/matrix-org/matrix-js-sdk).
+_Element Call_ needs to know the _SFU_ in order to connect to it.
+For this checkout the project with the current feature branch:
+
+```bash
+git clone https://github.com/vector-im/element-call.git
```

Comment:
You'll also want to yarn link the js-sdk

Reply:
Ok, i will change it like this.